cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/brokers/DatacenterBrokersMappingComparison.java

```diff
@@ -120,37 +118,39 @@
      * @param args
      */
     public static void main(String[] args) {
-        final long seed = 0;
-        final boolean showTables = false;
+        System.out.println("Starting comparison...");
+
+        final long seed = System.currentTimeMillis();
+        final boolean verbose = false;
 
         // Heuristic
         final CloudSim simulation0 = new CloudSim();
-        UniformDistr random0 = new UniformDistr(0, 1, seed);
+        final UniformDistr random0 = new UniformDistr(0, 1, seed);
         final DatacenterBrokerHeuristic broker0 = createHeuristicBroker(simulation0, random0);
-        new DatacenterBrokersMappingComparison(broker0, random0, showTables);
+        new DatacenterBrokersMappingComparison(broker0, random0, verbose);
 
         // BestFit
         final CloudSim simulation1 = new CloudSim();
-        UniformDistr random1 = new UniformDistr(0, 1, seed);
+        final UniformDistr random1 = new UniformDistr(0, 1, seed);
         final DatacenterBroker broker1 = new DatacenterBrokerBestFit(simulation1);
-        new DatacenterBrokersMappingComparison(broker1, random1, showTables);
+        new DatacenterBrokersMappingComparison(broker1, random1, verbose);
 
         // Simple - RoundRobin
         final CloudSim simulation2 = new CloudSim();
-        UniformDistr random2 = new UniformDistr(0, 1, seed);
+        final UniformDistr random2 = new UniformDistr(0, 1, seed);
         final DatacenterBroker broker2 = new DatacenterBrokerSimple(simulation2);
-        new DatacenterBrokersMappingComparison(broker2, random2, showTables);
+        new DatacenterBrokersMappingComparison(broker2, random2, verbose);
+
+        System.out.println("Comparison finished!");
     }
 
     /**
      * Default constructor where the simulation is built.
      */
-    public DatacenterBrokersMappingComparison(final DatacenterBroker brkr, final ContinuousDistribution rand, final boolean showTables) {
+    public DatacenterBrokersMappingComparison(final DatacenterBroker brkr, final ContinuousDistribution rand, final boolean verbose) {
```

Comment:
I think it's good to follow the same pattern used throughout the code, where parameters have the same name of the attributes they will be assigned to. This way, "brkr" will be "broker" and "broker = brkr" will be "this.broker = broker"